CalendarApp/NUnitTestProject/BookingServiceTests.DisplayTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Google.Apis.Calendar.v3.Data;
+using NUnit.Framework;
+using BookingRoom.Helpers;
+using Google.Apis.Calendar.v3;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests : BaseIntegrationTest
+    {
+        [Test]
+        public void BookingService_DisplayEvent_WrongCalendarIdThrowException()
```

Comment:
I thnk we should adopt our anming conventions. In cases when it is obvious that we are testing booking service(based on testfixture name) it makes sense to rename tests following convention
MethodTested_Conditions_ExpectedResult. Example DisplayEvent_WithWrongCalendarId_ThrowsException